presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -99,6 +99,14 @@ String
 
     Example type definitions: ``varchar``, ``varchar(20)``
 
+    Data insertion in VARCHAR supports usage of unicode literals with the prefix
```

Comment:
This syntax is not specifically related to data insertion. It can be used for a literal anywhere in any type of query.

Reply:
Updated